test/unit/streaming-client.spec.ts

```diff
@@ -133,6 +134,7 @@ describe('streaming-client', () => {
 
             // Assert
             expect(connectionError).toBe(expectedError);
+            expect(res.writable).toBe(false);
```

Comment:
Should we add an assertion to at one other valid test case for when `expect(res.writable).toBe(true);`